nevergrad/optimization/optimizerlib.py

```diff
@@ -1534,6 +1540,220 @@ def __init__(
 BO = ParametrizedBO().set_name("BO", register=True)
 
 
+class _HyperOpt(base.Optimizer):
```

Comment:
since the code is quite big and an interface to another lib, which may require even more integrations later on, let's move this to a different file (example "nevergrad/optimization/nghyperopt.py", I would at least avoid calling the file "hyperopt.py" because that could make a mess with the imports!)